examples/templates/docker-with-dotfiles/main.tf

```diff
@@ -28,24 +28,26 @@ data "coder_workspace" "me" {
 }
 
 data "coder_parameter" "docker_image" {
-  name        = "What Docker image would you like to use for your workspace?"
-  description = "The Docker image will be used to build your workspace."
-  default     = "codercom/enterprise-base:ubuntu"
-  icon        = "/icon/docker.png"
-  type        = "string"
-  mutable     = false
+  name         = "docker_image"
+  display_name = "What Docker image would you like to use for your workspace?"
+  description  = "The Docker image will be used to build your workspace."
+  default      = "codercom/enterprise-base:ubuntu"
+  icon         = "/icon/docker.png"
+  type         = "string"
+  mutable      = false
 }
 
 data "coder_parameter" "dotfiles_uri" {
-  name        = "What dotfiles repo would you like to use for your workspace?"
-  description = <<-EOF
+  name         = "dotfiles_uri"
+  display_name = "What dotfiles repo would you like to use for your workspace?"
```

Comment:
Display name should match name, so `dotfile URI`. You can move the question to the description.